### PR TITLE
Update bug template

### PR DESCRIPTION
**Pull request by precious-onyenaucheya-ons**
### What is the context of this PR?

Fixes https://github.com/ONSdigital/design-team/issues/109
This PR updates the existing bug template in the project. The labels, status and project was also updated in the template.

### How to review this PR

Check that the template preview matches the image in the issue linked

### Checklist

This needs to be completed by the person raising the PR.

-   [x] I have selected the correct Assignee
-   [x] I have linked the correct Issue

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -1,102 +1,82 @@
...
       attributes:
-          label: What was the expected outcome?
-      validations:
-          required: true
-    - type: textarea
-      id: impact
-      attributes:
-          label: How is this problem impacting your service?
+          label: Expected behaviour / outcome
+          description: 'If applicable, add screenshot(s) or examples of the expected behaviour'
```
Comment (rmccar) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:23]: "if applicable add screenshot(s) or examples of the bug/issue"
This is the wording on the card

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -1,102 +1,82 @@
...
+          label: Additional context
+          description: 'For example: describe the severity of the issue, any additional evidence, screenshots, proposed solution'
       validations:
           required: false
     - type: input
       id: version
       attributes:
           label: What version of the ONS Design System are you using?
-          placeholder: For example, 32.2.7
+          placeholder: 'For example: 32.2.7'
```
Comment (rmccar) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:43]: I think this should be a label rather than a placeholder

**Review by rmccar — CHANGES_REQUESTED** [review]
Couple of minor things

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 2 above)
Comment (alessioventuriniAND) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:43]: This needs to be a label and no longer a placeholder.

Comment (precious-onyenaucheya-ons) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: updated

**Review by rmccar — APPROVED** [review]

**Review by alessioventuriniAND — APPROVED** [review]

**Review by SriHV — APPROVED** [review]

**Pull request merged**